src/coreclr/jit/block.h

```diff
@@ -666,6 +669,38 @@ struct BasicBlock : private LIR::Range
         SetJumpKindAndTarget(jumpKind, jumpDest DEBUG_ARG(compiler));
     }
 
+    BasicBlock* GetFallThroughSucc() const
```

Comment:
Yes, in the case of `BBJ_COND` blocks, this would be the destination of the false branch. I wanted this name to capture the idea that, in the case of `BBJ_COND` blocks, `bbFallThroughSucc` will be taken if `bbJumpDest` isn't, and that this fall-through successor isn't necessarily the next block (which means the control flow doesn't actually "fall through"). Do you have any suggestions for the name @SingleAccretion? I don't want it to sound specific to `BBJ_COND` since we might decide to allow call-finally pairs to be non-contiguous later, thus making `bbFallThroughSucc` relevant.

Reply:
I would suggest `GetNormalJumpDest()` for symmetry with `GetJumpDest()`. It matches the expected codegen for `BBJ_COND`:
```
jcc <cond jump dest>
jmp <normal jump dest>
```

I wouldn't worry too much about the callfinally cases. Callfinally cases are inherently complex, one has to look up how they work specifically. Even if you make the `BBJ_ALWAYS` part of the pair non-contiguous with the callfinally, you'll still not have it as a "successor" in the flowgraph sense.

Reply:
I see, thanks for the suggestion! I'll change it.